Files/LUISApp.json

```diff
@@ -142,17 +142,17 @@
       "entities": []
     },
     {
-      "text": "list new idea",
+      "text": "list new ideas",
       "intent": "Idea.List",
       "entities": []
     },
     {
-      "text": "show me idea in progress",
+      "text": "show me ideas in progress",
       "intent": "Idea.List",
       "entities": []
     },
     {
-      "text": "show me new idea",
```

Comment:
This is not a typo. It was specially created for better compatibility in case that users may misspell it.
You can find "show me new ideas" on [line 120](https://github.com/OfficeDev/Office-365-Huddle-Templates/blob/master/Files/LUISApp.json#L120)

Reply:
Rolled back